### PR TITLE
kubectl druid plugin release-0.0.1

### PR DESCRIPTION
**Pull request by AdheipSingh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+- Scale Druid Replicas for a specific druid cr node in a namespace
+```
+kubectl druid scale --cr <cr> --namespace <namespace> --node middlemanager --replicas 4
+```
+
+- Update Image for a specific druid CR node in namespace
+```
+kubectl druid update --CR <CR> --image <image> --namespace <namespace> --node broker
```
Comment (nishantmonu51) [inline README.md]: nit: I believe --CR is case insensitive. so you can just lowercase it to be consistent with other places.

File: cmd/writers.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+type druidWritererCmd struct {
```
Comment (nishantmonu51) [inline cmd/writers.go]: rename to druidWriterCmd ?

File: cmd/writers.go [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+	f.StringVar(&node, "node", "", "name of druid node created by the druid operator, can be a statefulset or deployment")
+	f.StringVar(&cr, "cr", "", "name of the druid CR")
+	f.Int64Var(&replicas, "replicas", replicas, "number of replicas to scale")
+
+	return cmd
+}
+
+func (sv *druidListCmd) druidCRWriterNodeSpecReplicasRun(nodeName, namespace, CR string, replica int64) error {
+
+	writerResult, err := di.writerDruidNodeSpecReplicas(nodeName, namespace, CR, replica)
```
Comment (nishantmonu51) [inline cmd/writers.go]: nit: variable name CR -> cr

**Review by nishantmonu51 — APPROVED** [review]
👍 , LGTM, left very minor nit about some namings

**Comment by AdheipSingh** [comment]
@nishantmonu51 have done the changes.
Thanks

**Pull request merged**